src/components/modals/JoinSessionModal.svelte

```diff
@@ -12,15 +12,12 @@
     } from '@lib/jam/rooms';
     import { onMount } from 'svelte';
 
-    onMount(() => {
-        getJamRooms().then((value) => {
+    onMount(async () => {
+        await getJamRooms().then((value) => {
             rooms = value;
         });
     });
```

Comment:
We should use either `async/await` or Promises
```suggestion
    onMount(async () => {
       rooms =  await getJamRooms()
    });
```
or
```ts
     onMount(() => 
       // return the Promise
        getJamRooms().then((value) => {
            rooms = value;
        });
    );
    ```